src/test/components/Root-history.test.js

```diff
@@ -19,6 +19,13 @@ import { makeProfileSerializable } from '../../profile-logic/process-profile';
 
 import type { SerializableProfile } from 'firefox-profiler/types';
 
+// ListOfPublishedProfiles depends on IDB and renders asynchronously, so we'll
```

Comment:
I'm a bit nervous of excluding stuff from this file, as we have so little coverage on our history API.

Reply:
With this mock, we can assert that this panel is rendered (it will be rendered as `<list-of-published-profiles ...props>`, but we don't need to look at what's inside. I think that's the right amount of details for this test.